scene/main/viewport.cpp

```diff
@@ -1640,64 +1640,49 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 		if (mb->is_pressed()) {
 
 			Size2 pos = mpos;
```

Comment:
the problem with this is that, as long as a mouse button is pressed over a control (or left mouse button is, at least), all input should reach control, so preventing steal is right

Reply:
Ah, thanks Juan, I see what you mean. For instance, if I click and drag a visibility icon without letting up it will now start dragging a node when the mouse goes over it.

Can we just prevent focus stealing if it's the left mouse button then? I can definitely see someone accidentally hitting the middle and right mouse button in the correct order and wondering why their mouse is now "permanently" stuck to the viewport.

I just pushed up changes to reflect what I'm talking about.

Reply:
That updated code's also for illustration as things are still broken with it :)